export default {
  async addProblem() {
    try {
        
    } catch (error) {
      console.log(error.message);
    }
  },
};
